src/urdf_loader.h

```diff
@@ -24,16 +27,19 @@ namespace or_urdf
   {
   public:
     /** Opens a URDF file and returns a robot in OpenRAVE */
-    bool load(std::ostream& sout, std::istream& sin);
+    bool loadFile(std::ostream& sout, std::istream& sin);
+    bool loadJsonString(std::ostream& sout, std::istream& sin);
     
     /** Constructs plugin and registers functions */
     URDFLoader(OpenRAVE::EnvironmentBasePtr env) : OpenRAVE::ModuleBase(env)
     {
       __description = "URDFLoader: Loader that imports URDF files.";
       _env = env;
 
-      RegisterCommand("load", boost::bind(&URDFLoader::load, this, _1, _2),
+      RegisterCommand("load", boost::bind(&URDFLoader::loadFile, this, _1, _2),
```

Comment:
I agree with @cdellin. Lets:

1. Rename `URDFLoader::load` to `URDFLoader::loadModel`.
2. Rename `URDFLoader::loadFile` to `URDFLoader::loadURI`.
3. Rename the `Load` command to `LoadURI`.
4. Rename the `LoadJsonString` command to `LoadString`.
5. Add a `Load` command that prints a deprecation warning and forwards to `LoadURI`.